streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedTable.java

```diff
@@ -641,7 +641,7 @@
      * Aggregating is a generalization of {@link #reduce(Reducer, Reducer) combining via reduce(...)} as it,
      * for example, allows the result to have a different type than the input values.
      * If the result value type does not match the {@link StreamsConfig#VALUE_SERDE_CLASS_CONFIG default value
-     * serde} you should use {@link #aggregate(Initializer, Aggregator, Aggregator, Serde)}.
+     * serde} you should use {@link #aggregate(Initializer, Aggregator, Aggregator, Materialized)}.
```

Comment:
`Serde` is referenced elsewhere in the deprecated overloads. I think this is the only instance in `KGroupedTable` where a new overload still referenced `Serde`.

Reply:
LGTM.